webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
Looking back, I think it's right as it is (no change). The event should fire after the state change, but before any possible later state change.
(It took me a while to realize that "fire an event" means "call the event handler NOW", not "put an event in a queue and call the event handler later").